src/modules/elements/tooltip/style.pcss

```diff
@@ -0,0 +1,14 @@
+.components-tooltip {
```

Comment:
Should this one be namespace to affect only `tribe` items?

Reply:
The div is generated outside the containers

Reply:
As far as I see it's our wrapper for `WpTooltip`

Reply:
Do you think it might be a good idea to attach a class to the wrapper otherwise seems like a good fix for the general tooltip if other components uses it  with long text.

Reply:
https://github.com/WordPress/gutenberg/tree/master/packages/components/src/tooltip

The way the component is, there is no way to namespace or add a class to the tooltip.